src/renderer/data/ValidatorNetwork/ValidatorNetwork.tsx

```diff
@@ -64,11 +64,11 @@ function ValidatorNetwork() {
   let statusClass = 'text-danger';
   if (validator.status === NetStatus.Running) {
     statusText = 'Available';
-    statusClass = 'text-success';
```

Comment:
I'd much rather we use and over-ride the bootstrap & bootswatch css classes - ie, can we make `text-success` _be_ the green you want to use?

One of the reasons I did the conversion is to have something less invented by us.